kubeflow/README.md

```diff
@@ -0,0 +1,117 @@
+# Kubeflow Anywhere
+
+This document detals the steps needed to run the kubeflow project in different environments, such as Minikube (local laptop). Google Kubernetes Engine, etc. 
+
+## Quick Start
+
+In order to quickly setup all components of the stack, run:
+
+```commandline
+kubectl apply -f manifests/ -R
+```
+
+The above command sets up JupyterHub, an API for training using Tensorflow, and a set of deployment files for serivng. 
+Used together, these serve as configuration that can help a user go from training to serving using Tensorflow with minimal
+effort in a portable fashion between different environments. You can refer Instructions for using each of these components below. 
+
+## Minikube
```

Comment:
Do we need to have top level sections for minikube and GKE which then have the same subections (e.g. single node, distributed training).

The message we want to send is that the commands on minkube are the same as GKE.

So ideally we had have a single set of instructions for the various steps that should work regardless of the deployment.

For training this should work; i.e. to launch a training job the command should be

kubectl create -f tf_job.yaml



/cc @aronchick

Reply:
I considered that, and switched between layouts a bit. Open to suggestions.

Reply:
Done, changed layout to have less duplication.